extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -30,6 +30,9 @@ export class AmpSlideScroll extends BaseCarousel {
     /** @private @const {!Window} */
     this.win_ = this.getWin();
 
+    /** @private @const {boolean} */
+    this.hasLooping_ = this.element.hasAttribute('loop');
```

Comment:
At this point I also have to ask if you can see a better org for this where we ensure that this reimplementation and any other is complete. E.g. we currently also going to be missing `autoplay` and possibly some other parts of the protocol. Re-implementing `autoplay` in our class is not great, since 99% of that code should be reusable between this class and old one.